src/main/java/com/report/portal/business/page/objects/BasePage.java

```diff
@@ -0,0 +1,24 @@
+package com.report.portal.business.page.objects;
+
+import com.codeborne.selenide.Condition;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class BasePage {
+
+    public static BasePage initBase() {
+        return new BasePage();
+    }
+
+    public BasePage assertLogoVisible() {
+        $(By.className("layout__corner-area--DwLCT")).shouldBe(Condition.visible);
+        return this;
+    }
+
+    public static AllDashboardsPage gotoDashboardsPage() {
+        $(By.xpath("//a[contains(@href, 'dashboard')]")).click();
```

Comment:
I don't think I need to create so many variables. I store in variables only those selectors that I need to reuse. otherwise, it would be hard to change and understand.

Reply:
I agree with you.